app/models/miq_cockpit_ws_worker.rb

```diff
@@ -0,0 +1,92 @@
+require 'miq_apache'
+require 'runcmd'
+require 'linux_admin'
+
+class MiqCockpitWsWorker < MiqWorker
+  require_nested :Runner
+
+  APACHE_CONF_FILE = '/etc/httpd/conf.d/manageiq-cockpit.conf'.freeze
+  class_attribute(:web_service_uri)
+
+  self.web_service_uri = "http://localhost:4000"
+  self.required_roles = ['cockpit_ws']
+  self.maximum_workers_count = 1
+
+  def friendly_name
+    @friendly_name ||= "Cockpit Worker"
+  end
+
+  def self.can_start_cockpit_ws
+    if @supports_cockpit_ws.nil?
+      @supports_cockpit_ws = MiqEnvironment::Command.is_linux? && \
```

Comment:
Prefer `||=` over `if .nil?` and then having to return the value at the end separately.